### ISSUE TITLE
Provide GEMS support in GDB processing.

### ISSUE THREAD

**Issue by NoisyFlowers** [azgs/azlibrary_database#38]
Currently, gdb.js handles all flavors of gdb. The only flavors we've processed are NCGMP09. Need to verify this will work for GEMS, and make changes if necessary.

**Comment by aazaff** [comment]
@NoisyFlowers, 

The new expectation is that we will add another PostgreSQL schema to the database named `GEMS1`.

`GEMS1` will consist of the following featureclasses and non-spatial tables extracted from 
[GEMS.zip](https://github.com/azgs/azlibrary_database/files/4754926/GEMS.zip)

1. MapUnitPolys
2. ContactsAndFaults
3. DescriptionOfMapUnits
4. DataSources
5. Glossary
6. Stations
7. OrientationPoints

When `azlibadd` looks adds a new collection, it will look in gisdata/gems1. If it sees a a zipped or unzipped geodatabase (gisdata/gems1/mygdb.zip) then it should extract those tables from the geodatabase and insert them into the GEMS1 schema, and additionally include the relevant collection_id as a FOREIGN KEY.

In short, this should be implemented identically to https://github.com/azgs/azlibrary_database/tree/master/azlibConfigGDB but using the above zip file as the configuration.

p.s., can you send me an email so I know you received the notification of this issue from github

**Comment by aazaff** [comment]
You can use the following real gems gdb for testing.

[Blythe.gdb.zip](https://github.com/azgs/azlibrary_database/files/4754991/Blythe.gdb.zip)

**Comment by NoisyFlowers** [comment]
This is implemented in:
    azlibConfigGDB: 036917d8e06d14f997f3bf48735649753d683ef1, 795729f52a2f4e678b076039b594488b227ca664, 506600d520eecaeb905c19d9c937adf38f921d13, 3c3fd28b3f664978edb87bb99cc5550a81939240
    azlibAdd: 3fa3f627ee3f74ecb8b30711f2cb3c929c2031f2, ed4658b42ea2bbc49a74a084922e45179f45c2ea, 5e246cf95ad6ca9b3a86caa01406e231cd1385d6

Note that the gems schema and required directory is gems2.

This is deployed to dev.

End-to-end testing instructions are in azgs/azlibrary_webclient#3

**Comment by NoisyFlowers** [comment]
This, along with the sister issues in api and webclient, are now running on prod. 

Test in https://github.com/azgs/azlibrary_webclient/issues/3 all passed.
Test in https://github.com/azgs/azlibrary_api/issues/63 passed.
